federation/backends/collections.md

```diff
@@ -1,6 +1,40 @@
 # Collections
 
-If back-ends offer/mirror the same datasets, it is required to align the metadata.
+If back-ends offer/mirror the same datasets, it is required to align names and metadata.
+
+The following standardized names have been agreed upon:
+
+- SENTINEL2_L1C
+- SENTINEL2_L2A
+- SENTINEL1_GRD
+- SENTINEL3_OLCI_L1B
+
+These are all Copernicus Missions, and the standard names refer to the archives prepared and distributed by ESA. When it is not possible/desirable to use this name as collection id, a 'common_name' can be added next to the 'id' property that identifies the collection as a standard archive.
+
 For the following datasets an agreement has been achieved:
 
-...
\ No newline at end of file
+## Sentinel L2A
+
+The common name for this collection is 'SENTINEL2_L2A'. It refers to the L2A products generated using the Sen2Cor software, which can be configured compatible with the products generated by ESA. Note that products in the ESA archive have also been processed with different versions of Sen2Cor, so it is not possible to specify a very specific version or configuration of the processing chain. 
+
+
+### Bands
+
+Band names for spectral bands follow the Bxx naming convention in use by ESA. For instance: B01, B02, B03, B08, B8A, B12
+
+- `SCL` = the Sen2Cor sceneclassification band
```

Comment:
scene classification band (add space between scene and classification)

Reply:
done